skate/source/includes/_leaderboard.md

```diff
@@ -0,0 +1,218 @@
+`tag`     | no       | String |       | A leaderboard tag to filter on.
+`start_date` | no    | Date String | The beginning of time. | The start date for a date range filter.
+`end_date` | no      | Date String | Now | The end date for a date range filter.
+
+
+
+
+## Create a new score
+
+> Creating the score:
+
+```java
+import gamecenter
+client = GameCenterClient("myAPIKey");
+
+int user_id = 1;
+int score = 220;
+String tag = "level two";
+
+Score s = client.newScore(user_id, score, tag);
+```
+
+```shell
+curl "https://tmwild.com/api/leaderboards?user_id=1&score=220&tag=level two"
+  -X POST
+  -H "Authorization: myAPIKey"
+```
+
+> Example Response:
+
+```json
+{
+  "id": 5,
+  "user_id": 1,
+  "score": 220,
+  "tag": "level two",
+  "created_at": "2015-07-12T12:16:22"
+}
+```
+
+This endpoint creates a new score in the leaderboard and returns the score object back upon success.
+
+### Query Parameters
+
+Parameter | Required | Type | Description
+--------- | -------- | ---- | -----------
+`user_id` | yes      | Integer | The ID of the user to create a leaderboard entry for.
+`score`   | yes      | Integer | The score for the entry in the leaderboard.
+`tag`     | no       | String | An identification tag for a leaderboard entry.
+
+
+
+
+## Create a new score and list nearby scores
+
+> Creating the score:
+
+```java
+import gamecenter
+client = GameCenterClient("myAPIKey");
+
+int user_id = 1;
+int score = 220;
+String tag = "level two";
+int radius = 2;
+
+ArrayList<Score> scores = client.newScoreAndList(user_id, score, tag, radius);
+```
+
+```shell
+curl "https://tmwild.com/api/leaderboards?user_id=1&score=220&tag=level two&radius=1"
+  -X POST
+  -H "Authorization: myAPIKey"
+```
+
+> Example Response:
+
+```json
+[
+  {
+    "id": 53,
+    "user_id": 1,
+    "score": 222,
+    "tag": "level two",
+    "created_at": "2015-03-12T17:12:22"
+  }, {
+    "id": 5,
+    "user_id": 1,
+    "score": 220,
+    "tag": "level two",
+    "created_at": "2015-03-14T18:26:22"
+  }, {
+    "id": 5,
+    "user_id": 1,
+    "score": 219,
+    "tag": "level two",
+    "created_at": "2015-07-11T17:46:22"
+  }
+]
+```
+
+Add a new score and receive the scores above and below the new score.
+
+### Query Parameters
+
+Parameter | Required | Type | Default | Description
+--------- | -------- | ---- | ------- | -----------
+`user_id`   | yes    | Integer |      | The ID of the user to create a leaderboard entry for.
+`score`     | yes    | Integer |      | The score value for the entry in the leaderboard.
+`radius`    | yes    | Integer | `4`  | The number of scores to return above and below the user's score.
+`sort`    | no       | String | `"descending"` | The order the results will be returned, either `"ascending"` or `"descending"`.
+`tag`       | no     | String |       | An identification tag for a leaderboard entry.
+`filter_tag` | no    | String |       | A leaderboard tag to filter on. Must be either empty or the same as `tag`.
```

Comment:
`filter_tag` functions the same as `tag` in the other GET endpoints: it filters the returned scores. I imagine a scenario where you make a new score for level 2, so you have `tag` = `"level 2"`, but you want to display the cross-level scores, so you have `filter_tag` = `""`.

It has to have that name because `tag` is taken by the creation part of this request.